js/base-weather.js

```diff
@@ -178,7 +178,9 @@ function baseWeather () {
 		function getTintColor (page) {
 			const tintEnabled = page.get("bR20cfg_weatherTint1");
 			if (tintEnabled) {
-				return `${(page.get("bR20cfg_weatherTintColor1") || "#4c566d")}80`;
+				const tintOpacity = page.get("bR20cfg_weatherTintOpacity1") || d20plus.weather.props.weatherTintOpacity1;
+				const tintOpacityHex = tintOpacity ? (14 + Math.round((243 - 16) * tintOpacity)).toString(16) : 80;
```

Comment:
what's all this `(14 + Math.round((243 - 16) * tintOpacity)).toString(16)` about